tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -449,11 +449,12 @@ stages:
               --security-opt seccomp=unconfined \
               --volume $(Build.SourcesDirectory):/onnxruntime_src \
               --volume $(Build.BinariesDirectory):/build \
+              --entrypoint /bin/bash \
```

Comment:
this is interesting. Why --entrypoint as opposed to calling shell directly?

Reply:
@suffiank Is this change needed?